cpp/unreal_projects/SpearSim/Config/DefaultEngine.ini

```diff
@@ -27,11 +27,11 @@ UnrealEdEngine=/Script/SpearSimEditor.SpearSimEditorUnrealEdEngine
 ; Configure Unreal's physics simulation parameters to achieve a reasonable tradeoff between fidelity and performance.
 ; These parameters can be overridden via our config system, but we leave them here also, because editor sessions might
 ; not reach the point where we load the values from our config system.
-[/Script/Engine.PhysicsSettings]
```

Comment:
Revert all changes to `[/Script/Engine.PhysicsSettings]` since you probably did not intend to make any changes here.